consensus/poet/cli/sawtooth_poet_cli/registration.py

```diff
@@ -40,13 +40,32 @@
     VR_NAMESPACE + sha256('validator_map'.encode()).hexdigest()
 
 
-def add_genesis_parser(subparsers, parent_parser):
-    """Add argument parser arguments for the `poet genesis` subcommand.
+def add_registration_parser(subparsers, parent_parser):
+    """Add argument parser arguments for the `poet registration`
+    subcommand
+    """
+    description = 'Provides subcommands for creating PoET registration'
+
+    parser = subparsers.add_parser(
+        'registration',
+        help=description,
+        description=description + '.')
+
+    registration_sub = parser.add_subparsers(
+        title='subcommands', dest='subcommand')
+    registration_sub.required = True
+
+    add_create_parser(registration_sub, parent_parser)
+
+
+def add_create_parser(subparsers, parent_parser):
+    """Add argument parser arguments for the `poet registration create`
+    subcommand.
     """
     description = 'Creates a Batch for the genesis block'
```

Comment:
Sorry @nick-drozd I missed this the first time through. Fine by me if you want to merge this PR and create a separate fix for this help message - or just do the fixup in this PR.
Should be updated like the later help message .. remove 'genesis' and add text about validator registry.